static/js/Hello.jsx

```diff
@@ -1,6 +1,10 @@
 import React from "react";
 import { Button, Grid, Row, Col } from "react-bootstrap";
 
+const language = 'en-us'
+const text = 'something to say'
+const url = `${origin}/${language}/${text}`
+
```

Comment:
I think we want these to be defined inside `getPythonHello()`, probably we should replace `text` in this case with `this.state.greeting` is that correct ? If so then we need to add `language` to the component state too and probably add a `select` component to display the available languages and handle changing the language 🤔

Reply:
I think i will reset the frontend, and build something new from scratch to better match the project.
That was my idea too, add a drop down menu with the available languages and return the selected one + text box + button + some styling.

Reply:
Aha so you'll handle the front-end in another PR ?

Reply:
No, just clean this up, there's "junk" from another project, the getPythonHello, greeting, and hello is from another small project i made to test communication between ReactJS and Flask, and i refurbished it to work this new backend, but there are too many loose ends that don't match this backend. 

I really need to start with the front-end, but i am doing a portfolio app with ReactJ and i need to finish it first. 
I will upload it later to github for you to take a look at and give your saying... :)